pythonforandroid/bootstraps/common/build/src/main/java/org/kivy/android/PythonService.java

```diff
@@ -90,13 +96,13 @@ public int onStartCommand(Intent intent, int flags, int startId) {
     protected void doStartForeground(Bundle extras) {
         String serviceTitle = extras.getString("serviceTitle");
         String serviceDescription = extras.getString("serviceDescription");
-
         Notification notification;
         Context context = getApplicationContext();
         Intent contextIntent = new Intent(context, PythonActivity.class);
         PendingIntent pIntent = PendingIntent.getActivity(context, 0, contextIntent,
             PendingIntent.FLAG_UPDATE_CURRENT);
-        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.HONEYCOMB) {
+
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.O) {
```

Comment:
Why are we not checking against `HONEYCOMB`? Is it on purpose or by mistake?

Reply:
that was on purpose. It is true that HONEYCOMB(value:11)  and O(value:26) stand for Android 8, but only O can be used to query for version ranges.
see [Build.java](https://android.googlesource.com/platform/frameworks/base/+/master/core/java/android/os/Build.java).
I have tried it and if I use HONEYCOMB for coparison it fails on a device runnng android 7